blueman/main/applet/BluezAgent.py

```diff
@@ -94,21 +91,11 @@ def on_insert_text(editable: Gtk.Entry, new_text: str, _new_text_length: int, _p
 
         return dialog, pin_entry
 
-    def get_device_string(self, device_path: str) -> str:
-        device = Device(obj_path=device_path)
+    def get_device_string(self, object_path: str) -> str:
+        device = Device(obj_path=object_path)
         return f"<b>{escape(device.display_name)}</b> ({device['Address']})"
 
-    @overload
-    def ask_passkey(self, dialog_msg: str, is_numeric: "Literal[True]", device_path: str, ok: Callable[[int], None],
-                    err: Callable[[Union[BluezErrorCanceled, BluezErrorRejected]], None]) -> None:
-        ...
-
-    @overload
-    def ask_passkey(self, dialog_msg: str, is_numeric: "Literal[False]", device_path: str, ok: Callable[[str], None],
-                    err: Callable[[Union[BluezErrorCanceled, BluezErrorRejected]], None]) -> None:
-        ...
-
-    def ask_passkey(self, dialog_msg: str, is_numeric: bool, device_path: str, ok: Callable[[Any], None],
+    def ask_passkey(self, dialog_msg: str, is_numeric: bool, object_path: str, ok: Callable[[Any], None],
```

Comment:
The overloads make sense as they restrict the arguments to either `is_numeric = true` and a `Callable[[int], None]` or `is_numeric = false` and a `Callable[[str], None]`. Without, any `Callable[[Any], None]` is fine independent of `is_numeric`.

Reply:
I completely missed the second part that is different. I'll back this out.